docker-compose.dev.yml

```diff
@@ -68,7 +68,9 @@ services:
       - "--stripe-secret-key=${STRIPE_SECRET_KEY}"
       # used only for local development
       - "--jwt-signing-private-key=LS0tLS1CRUdJTiBQUklWQVRFIEtFWS0tLS0tCk1DNENBUUF3QlFZREsyVndCQ0lFSUR5V0ZFYzhKYm05NnA0ZGNLTEwvQWNvVUVsbUF0MVVKSTU4WTc4d1FpWk4KLS0tLS1FTkQgUFJJVkFURSBLRVktLS0tLQo="
-      - "--permit-api=https://api.eu-central-1.permit.io"
+      - "--permit-api-uri=https://api.eu-central-1.permit.io"
```

Comment:
I imagine this can differ based on how a Permit account is configured. Makes me think that it is better to hide it behind an env var.